paper-ieee-trans.tex

```diff
@@ -248,26 +250,26 @@ \subsection{Enabling Grids for E--sciencE}
 
 The three phases of Enabling Grids for E--sciencE ({\sl EGEE}, Apr 2004 -- Apr 2010)
 \cite{cordis:egee, cordis:egee2, cordis:egee3} projects brought together
-scientists and engineers from more than 240 institutions in 45 countries to
+scientists and engineers from over 240 institutions in 45 countries to
 provide a seamless Grid infrastructure for e--Science. {\sl EGEE--II} and {\sl EGEE--III}
-featured the internationalization of the project, embracing worldwide research
+featured the internationalization (outside Europe) of the project, embracing worldwide research
 institutions and user communities. The software to sustain the increasing
 requirements from the diverse scientific communities would need to develop a
 rich set of new services while maintaining a sustainable infrastructure for
 Grid computing, eventually used by more than 15 thousand researchers and deployed in
 over 250 institutions.
 
-The {\sl gLite} middleware \cite{glite} was the ultimate
+The {\sl gLite} middleware \cite{glite} was the
 official software distribution of {\sl EGEE} as of 2006, after two years of prototyping and
-re--engineering efforts to converge with {\sl LHC} Computing Grid (LCG--2), Virtual
-Data Toolkit (VDT) and Condor \cite{condor} software distributions. The
+re--engineering efforts to converge with the {\sl LHC} Computing Grid (LCG--2), Virtual
+Data Toolkit (VDT) and Condor \cite{condor} software stacks. The
 development team was comprised of more that 80 people from 12 academic and
-industrial partners, that issued more than 10 thousand bug fixes, 1.7 thousand patches and
+industrial partners, fixing more than 10 thousand bug, 1.7 thousand patches and
```

Comment:
we don't know if all of them were fixed ;)
Bugs issued (solved or not) is a good estimation of the work done. Anyway if leaving like you propose, correct "bug" with "bugs"